app/src/main/java/org/dieschnittstelle/mobile/android/skeleton/viewmodel/TaskListViewModel.java

```diff
@@ -90,5 +90,15 @@ public void updateTask(Task taskFromDetailView) {
     }
 
     public void deleteTask(long id) {
+        processingState.setValue(ProcessingState.RUNNING);
+        executorService.execute(() -> {
+            Task task = new Task();
+            task.setId(id);
+            boolean isDeleted = taskDbOperation.deleteTask(id);
+            if (isDeleted) {
+                getTaskList().remove(task);
```

Comment:
hier scheint es auch so, dass es versucht, ein neues definiertes Task in Zeile 95 zu löschen und damit wird hier keine gelöscht, weil das Task nicht in TaskList existiert

Reply:
Wie eben